sstables/proceed_generator.hh

```diff
@@ -24,24 +24,21 @@
 #include <seastar/core/coroutine.hh>
 #include "sstables/consumer.hh"
 
-
-
-
 class proceed_generator {
 public:
     struct promise_type {
-        using handle_type = SEASTAR_INTERNAL_COROUTINE_NAMESPACE::coroutine_handle<promise_type>;
+        using handle_type = std::experimental::coroutine_handle<promise_type>;
```

Comment:
Unless I'm missing something, you added this code in a previous patch. So please merge the fix into it.

Reply:
I will avoid SEASTAR_INTERNAL_COROUTINE_NAMESPACE in the new PR altogether